### DISCUSSION TITLE
下载测速为0问题

### DISCUSSION THREAD

**Discussion by jeasonliang** [XIU2/CloudflareSpeedTest#457] [讨论]
### 问题描述

前段时间稳定 ，最近下载测速为0，具体步骤如下：
1.下载zip.baipiao.eu.org的文件 挑选443和8443的ip地址到cloudflare项目下的 ip.txt文档；
2.开启测速条件 测速地址用cloudflare自建的和https://github.com/XIU2/CloudflareSpeedTest/issues/168提供的都不行 速度均为0，如下图：
![Snipaste_2023-11-04_11-04-47](https://github.com/XIU2/CloudflareSpeedTest/assets/27670071/924ba29d-4462-4d20-be53-55cd6abb9982)

不知道是哪里出错了 ，我测速时候关闭了代理！有没有相同情况的？

### 软件版本

v2.24

**Comment by XIU2** [comment]
非软件问题，我无法解决。

这种情况，要么是 IP 问题、要么是下载测速地址问题、要么就是你的网络问题，你只能一个个排除。

这种情况是前几个月开始集中出现的，已经有不少人反馈过，大概率是各地对 Cloudflare 的干扰阻断加强了？

**Reply by jeasonliang** [reply]
感谢指导！谢谢！

**Comment by shylock-wu** [comment]
虽然下载速度为0，但是光看延迟的话，选择的ip也能使用😂
